Missing <cstdint> header in urdf_parser.h

`urdf_parser.h` uses `uint32_t`, so it's missing the cstdint header:

```
--- urdf_parser.h  2025-04-29 13:32:39.494394017 +0200
+++ urdf_parser.h   2025-04-29 13:32:51.395458874 +0200
@@ -37,6 +37,7 @@
 #ifndef URDF_PARSER_URDF_PARSER_H
 #define URDF_PARSER_URDF_PARSER_H
 
+#include <cstdint>
 #include <stdexcept>
 #include <string>
 #include <vector>
```

Otherwise, with g++-15:
```
/usr/include/urdfdom/urdf_parser/urdf_parser.h:97:14: error: 'uint32_t' has not been declared
```

--- a/urdf_parser/include/urdf_parser/urdf_parser.h
+++ b/urdf_parser/include/urdf_parser/urdf_parser.h
@@ -37,6 +37,7 @@
 #ifndef URDF_PARSER_URDF_PARSER_H
 #define URDF_PARSER_URDF_PARSER_H
 
+#include <cstdint>
 #include <stdexcept>
 #include <string>
 #include <vector>

